.rubocop_todo.yml

```diff
@@ -500,11 +464,6 @@ Style/RegexpLiteral:
 Style/RescueEnsureAlignment:
   Enabled: false
 
-# Offense count: 46
```

Comment:
From my perspective, this is the only from this PR with potential quality impact on the project.